apps/jobs/sender.py

```diff
@@ -72,10 +74,40 @@ def publish_submission_message(message):
         challenge = Challenge.objects.get(pk=message["challenge_pk"])
     except Challenge.DoesNotExist:
         logger.exception(
-            "Challenge does not exist for the given id {}".format(message["challenge_pk"])
+            "Challenge does not exist for the given id {}".format(
+                message["challenge_pk"]
+            )
         )
         return
     queue_name = challenge.queue
+    slack_url = challenge.slack_webhook_url
     queue = get_or_create_sqs_queue(queue_name)
     response = queue.send_message(MessageBody=json.dumps(message))
+
+    # sending slack notification
+    if slack_url:
+
+        challenge_name = challenge.title
+        submission = Submission.objects.get(pk=message["phase_pk"])
```

Comment:
I think `phase_pk` is wrong here. It should be `message["submission_pk"]`.

Reply:
Also, can you please use `get_submission_model` function here.
Ref: https://github.com/Cloud-CV/EvalAI/blob/master/apps/jobs/views.py#L910